pycaption/scc/__init__.py

```diff
@@ -232,6 +232,22 @@ def read(self, content, lang='en-US', simulate_roll_up=False, offset=0):
         captions = CaptionSet({lang: self.caption_stash.get_all()})
 
         # check captions for incorrect lengths
+        lines = []
```

Comment:
this could be a separate method, the read method is already very crowded and it would be easier to read if instead of a new code block you would use a method call